include/boost/geometry/algorithms/detail/equals/interface.hpp

```diff
@@ -106,13 +113,37 @@ struct equals
                 Geometry1, Geometry2
             >::apply(geometry1, geometry2, strategy);
     }
+};
+
+template <typename Strategy>
+struct equals<Strategy, false>
+{
+    template <typename Geometry1, typename Geometry2>
+    static inline bool apply(Geometry1 const& geometry1,
+                             Geometry2 const& geometry2,
+                             Strategy const& strategy)
+    {
+        //using strategies::equals::services::strategy_converter;
```

Comment:
is this going to used in the future? why do you use `relate` instead now?

Reply:
Probably not. I'll probably leave relate here. I'm using relate everywhere to avoid creating an even bigger PR.